### ISSUE TITLE
Add new booru cogs

### ISSUE THREAD

**Issue by Alzarath** [Alzarath/Booru-Cogs#1]
Sites that meet rating criteria and may be added
- [ ] All Girl
- [x] Danbooru
- [x] Derpibooru
- [ ] DrunkenPumken
- [x] e621
- [x] Gelbooru
- [x] Lolibooru
- [x] Konachan
- [ ] Yandere

Sites that will not be added due to the site/community lacking good rating tagging
- Equibooru
- Furrybooru
- Realbooru
- Rule 34

Feel free to suggest sites.

**Comment by ghost** [comment]
Could add yande.re, konachan.co, rule32.xxx, furry.booru.org, allgirl.booru.org to name some of the largest ones.. Found them on http://booru.org/top if you want to find more.

**Comment by Alzarath** [comment]
Thanks for the recommendations. I have a standard set by the Red-DiscordBot approval system where I will not implement boorus that can display explicit images without the user opting in. I measure this chance by simply searching for a mixture of tags including "rating:safe" and a number of tags that I believe belong in the explicit rating.

That being said, I assume by rule32, you mean rule34, as I could not find a rule32 anywhere. Neither **Rule34** nor **Furrybooru** meet these standards by a large margin.

**Konachan** passes with flying colors. **Yande.re** passes well enough. **All Girl** passes; it could do better, though.

As for the other parts of the top 5 from the site you linked, **Realbooru** fails pretty horribly. **Equibooru** just barely fails by one image from what I could tell, but it's an image that's been there for about 5 years, which does not give a good impression. **DrunkenPumken** passes pretty nicely.

I'll consider adding the ones that passed in the future.

**Comment by ghost** [comment]
Yeah, I meant 34, sorry about that. Thanks for this cog and good luck! :)

**Comment by ibigfire** [comment]
If someone is activating a rule34 cog on their server they are clearly opting in to wanting rule34 content.  Something seems off if the approval system doesn't have a way of allowing cogs that let bot owners put what they clearly intend to include and desire on their servers.  Any idea where the best place is to go to try to get this flaw in the system addressed?
